docs/developers/protocol/background/taker-compensation.md

```diff
@@ -7,39 +7,39 @@ Since Mangrove offers do not provision liquidity, there must be a mechanism that
 
 Mangrove uses [Offer Provisions](../technical-references/reactive-offer/offer-provision.md) as a protection mechanism.
 
-Consider an offer that promises 100WETH for 100DAI and requires 300k for execution. That gas will be paid for by the taker. If the 100WETH are delivered, all is well.
+Consider an offer that promises 100 WETH for 100 DAI and requires 300k for execution. That gas will be paid for by the taker. If the 100 WETH are delivered, all is well.
```

Comment:
Not sure but 100 WETH *is* delivered feels more natural to say to me

Reply:
Makes sense 👍 Addressed in 70653a03107bf5e8de37ed6f5609ab6b46a8cde1